extensions/metrics/README.md

```diff
@@ -0,0 +1,253 @@
+# @loopback/extension-metrics
+
+This module contains a component that reports metrics of Node.js, LoopBack
+framework, and your application to [Prometheus](https://prometheus.io/).
+
+## Stability - Experimental
```

Comment:
Let's use colon `:` to delimit the field name "Stability" from the value "Experimental. 

```suggestion
## Stability: Experimental
```

See e.g. Node.js docs (they use `:` after "Stability" and then `-` to delimit stability index from the description).

<img width="269" alt="Screen Shot 2019-09-19 at 16 36 10" src="https://user-images.githubusercontent.com/1140553/65253867-a74d0580-dafb-11e9-9121-f8838d613bdb.png">